index.html

```diff
@@ -5,9 +5,33 @@
 	<meta http-equiv="X-UA-Compatible" content="IE=edge">
 	<meta name="viewport" content="width=device-width, initial-scale=1.0">
 	<title>Wind D Bnb</title>
+	<script>
+		const db_local = 'stays.json'
+		const db_url = 'https://my-json-server.typicode.com/pamdayne/wind-d-bnb/'
```

Comment:
Is this expected?

![CleanShot 2022-04-18 at 14 06 05](https://user-images.githubusercontent.com/97009796/163763079-7b129609-47da-4043-8b2a-f57031c9f0e8.png)

Reply:
I think it's because i moved/renamed the file. i'll readd it back for you to see.

Reply:
@xv-yuyu-s , `db.json` is moved into root at `main` branch. you should be able to see it now.